tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -28,6 +42,34 @@ pub struct Layer<
 
 /// A builder for [`Layer`](struct.Layer.html) that logs formatted representations of `tracing`
 /// events and spans.
+///
+/// ```rust
+/// use tracing_subscriber::{
+///     fmt::{self, format::Format, time},
+///     layer::Layer as _,
+///     registry::Registry,
+///     EnvFilter,
+/// };
+///
+/// let default_filter = concat!(module_path!(), "=info");
+///
+/// let fmt = Format::default().with_timer(time::Uptime::default());
```

Comment:
maybe a comment saying that we're customizing the way that events will be formatted? 

or, i might split the examples into something like:
```rust
/// ## Examples
///
/// Constructing a layer with the default configuration:
/// ```rust
/// let fmt_layer = fmt::Layer::builder().finish();
/// ```
///
/// Overriding the layer's behavior:
///
/// ```rust
/// let fmt_layer = fmt::Layer::builder()
///    .with_target(false) // don't include event targets when logging
///    .finish();
/// ```
///
/// Setting a custom  event formatter:
///
/// ```rust
/// let fmt = Format::default().with_timer(time::Uptime::default());
/// let fmt_layer = fmt::Layer::builder()
///     .event_format(fmt)
///     .with_target(false)
///     .finish();
/// ```
```